open-vsx/publish-extensions
# Add reporting for most popular Marketplace Extensions

## Description

This PR adds a script which reports stats about the most popular extensions from MS which we don't have on OpenVSX, maybe pointing us to extensions which we don't have from Microsoft or which we should ask their maintainers about.

It also adds docs about understanding the reports that the GitHub Action generates as well as some quality of life improvements in the publishing and reporting scripts.

## How to test

- For the docs part, check it out and read through it
- For the reporting part, check the [workflow run](https://github.com/open-vsx/publish-extensions/actions/runs/2383294147) to see what it produces
